config/rails.yml

```diff
@@ -0,0 +1,21 @@
+Rails:
+  Enabled: true
+
+Rails/FindEach:
+  Enabled: false
+
+Rails/OutputSafety:
+  Enabled: true
+
+Rails/PluralizationGrammar:
+  Enabled: true
+
+Rails/RequestReferer:
+  Enabled: true
+  EnforcedStyle: referrer
+
+Rails/ScopeArgs:
+  Enabled: true
+
+Rails/UniqBeforePluck:
+  Enabled: true
```

Comment:
Yeah, we should move the `ApplicationRecord` linter to this list.

Though, I'm unsure about the naming convention. I'm a little worried about putting things in the `Rails/` namespaces as it could collide up upstream Cops. It'd be really cool if `GitHub/Rails/ApplicationRecord` worked. But I'm not sure if you can have nested namespaces. Let me actually test that though, cause it seems ideal.